app/views/parks/show.html.erb

```diff
@@ -53,3 +51,9 @@
   <%end%>
 
 </ul>
+
+<div>
+
+
+<%= render "layouts/review_form" %>
```

Comment:
This form should live in `app/views/reviews` and it should be named `_form.html.erb` Thus you should `render 'reviews/form'